### PR TITLE
fix(fmg): fix ordering and remove p-value

### PR DESCRIPTION
**Pull request by seve**
- #3801 #3804 

### Reviewers
**Functional:** @tihuan @atarashansky 

----

Changed backend repsonse shape to have `marker_genes` be an array to preserve ordering
Removed p-values from table

[RDEV](https://ordered-genes-frontend.rdev.single-cell.czi.technology/gene-expression?fmg=true)

### CODE REVIEW

File: backend/wmg/api/wmg-api.yml [hunk 1]
```diff
@@ -372,30 +364,24 @@ paths:
                   snapshot_id:
                     $ref: "#/components/schemas/wmg_snapshot_id"
                   marker_genes:
-                    type: object
-                    # we use `additionalProperties` instead of `properties`, since the object's property names are
-                    # ontology term ids, rather than a fixed set of names
-                    additionalProperties:
```
Comment (seve) [inline backend/wmg/api/wmg-api.yml:378]: GitHub's formatting on this diff is attrotious, sorry

File: backend/wmg/api/wmg-api.yml [hunk 2]
```diff
@@ -372,30 +364,24 @@ paths:
...
+                        Object with gene id and effect size.
                       type: object
                       properties:
-                        p_value:
-                          description: adjusted p-value
-                          type: number
-                          format: float
-                          maxLength: 4
-                          minimum: 0.0
-                          maximum: 1.0
```
Comment (seve) [inline backend/wmg/api/wmg-api.yml:390]: Oops this should still be here, fixing

**Review by atarashansky — CHANGES_REQUESTED** [review]
Looks good, but I think unit tests need to be updated to expect the new response format (a list instead of a dictionary).

`QueryTest.test__query_marker_genes_cube__returns_correct_top_10_markers`
`QueryTest.test__query_marker_genes_cube__returns_correct_all_markers`
`MarkerGeneCalculationTest.test__get_markers_ttest`
`MarkerGeneCalculationTest.test__get_markers_binomtest`
`WmgApiV1Tests.test__markers_returns_200_and_correct_response`
`WmgApiV1Tests.test__markers_returns_200_and_empty_dictionary_for_bad_celltypes`

File: frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx [hunk 3]
```diff
@@ -102,9 +102,6 @@ function CellInfoSideBar({
               </CopyGenesButton>
             </GeneHeaderWrapper>
           </GeneCellHeader>
-          <CellHeader hideSortIcon horizontalAlign="right">
```
[the remark this answers is not in the document]
Comment (seve) [reply frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx:105]: since they're already in git history I'm going to leave them as deleted

**Review by atarashansky — CHANGES_REQUESTED** [review]
Hey @seve - the tests are running now. PTAL, I think there are a couple of minor issues (like using assertDictEquals to compare lists - for lists, you can use assertEqual).

EDIT: I went ahead and fixed it - very minor changes required!

**Pull request merged**